partner_communication_switzerland/models/res_users.py

```diff
@@ -64,38 +64,62 @@ def _compute_signature(self):
                 "en_US": "https://www.facebook.com/compassionsuisse/",
             }
             lang = self.env.lang or self._context.get("lang") or self.env.user.lang
-            base_url = self.env["ir.config_parameter"].sudo().get_param("web.base.url")
 
             for user in self:
                 employee = user.employee_ids[:1].with_context(bin_size=False)
-                employee_image_url = f"{base_url}/employee/image/{employee.id}"
 
-                values = {
-                    "name": f"{user.preferred_name} {user.lastname}"
-                    if user.firstname
-                    else _("The team of Compassion"),
-                    "email": user.email if user.firstname else "info@compassion.ch",
-                    "lang": lang,
-                    "lang_short": lang[:2],
-                    "team": _("and the team of Compassion") if user.firstname else "",
-                    "job_title": employee.job_title or "",
-                    "office_hours": _("mo-thu: 9am-2pm"),
-                    "company_name": user.company_id.address_name,
-                    "phone_link": phone_link.get(lang),
-                    "phone": phone.get(lang),
-                    "mobile": employee.mobile_phone,
-                    "mobile_link": (employee.mobile_phone or "")
-                    .replace(" ", "")
-                    .replace("(0)", ""),
-                    "facebook": facebook.get(lang),
-                    "employee_image_url": employee_image_url,
-                }
+                if not employee:
```

Comment:
```suggestion
                if employee:
```

Reply:
oops forgot to revert this condition, was used for testing